### PR TITLE
Populate pieces on board when new game is created

### PR DESCRIPTION
**Pull request by AlekstheRealist**
Adjusted the Game model and wrote initialize_board! method/callback that populates the board with each of the pieces in each of their respective positions/coordinates. The initialize_board! method/callback is initiated after a game has been created.

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -1,4 +1,53 @@
 class Game < ActiveRecord::Base
   belongs_to :users
   has_many :pieces
+  after_create initialize_board!
+
+  def initialize_board!
+    #white pieces
+    7.times do |n|
+        Pawn.create(
+            piece_id: id,
```
Comment (lindyrp) [inline app/models/game.rb]: Should this be game_id to tie the pieces to the game it belongs to?

File: app/models/game.rb [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+            color: true,
+            y_cord: 1,
+            x_cord: n,
+            type: pawn
```
Comment (lindyrp) [inline app/models/game.rb]: Wouldn't the type be determined by the model itself? Do we need to specify this explicitly here?

File: app/models/game.rb (hunk 1 above)
Comment (AlekstheRealist) [inline app/models/game.rb]: You're right, I'll change "id" to "game_id" asap

File: app/models/game.rb (hunk 2 above)
Comment (AlekstheRealist) [inline app/models/game.rb]: Good point, I'm going to refactor this and double check with Ken pertaining to where we should specify it

**Pull request closed without merging**